crates/astria-sequencer/accounts-genesis.csv

```diff
@@ -0,0 +1,4 @@
+"address","balance"
+"alice","1000000000000000000"
```

Comment:
CSV files are very restrictive. I would prefer letting this be a json.

Reply:
yeah I prefer json also, because there might be other things in the initial app state other than balances, so it'd be easier to separate them by module